ios/AppLovinMAX.m

```diff
@@ -527,6 +604,14 @@ - (instancetype)init
         return;
     }
     
+    self.sdk.targetingData.yearOfBirth = nil;
+    self.sdk.targetingData.gender = ALGenderUnknown;
+    self.sdk.targetingData.maximumAdContentRating = ALAdContentRatingNone;
+    self.sdk.targetingData.email = nil;
+    self.sdk.targetingData.phoneNumber = nil;
+    self.sdk.targetingData.keywords = nil;
+    self.sdk.targetingData.interests = nil;
+
```

Comment:
`[self.sdk.targetingData clearAll];` will clear all targeting data under-the-hood already, so no need for these lines.

```suggestion
```

Reply:
Removed.  But, `clearAll` clears the data for sending to the server, then each field still returns the same value.  Could you check how the SDK handles each field to clear?